exasol_data_science_utils_python/utils/data_classes_runtime_type_check.py

```diff
@@ -5,6 +5,7 @@
 
 def check_dataclass_types(datacls):
     for field in fields(datacls):
-        typeguard.check_type(value=datacls.__dict__[field.name],
-                             expected_type=field.type,
-                             argname=field.name)
+        typeguard.check_type(
+            value=datacls.__dict__[field.name],
+            expected_type=field.type,
+        ) # removed argname=field.name
```

Comment:
Argument `argname` had been deprecated and has been removed in typeguard version 4.

Reply:
Ah thanks, see next push for fixed impl.